staking/programs/staking/src/lib.rs

```diff
@@ -751,6 +754,157 @@ pub mod staking {
         Ok(())
     }
 
+    pub fn slash_account(
```

Comment:
It shouldn't be possible to slash the governance target

Reply:
Done